lib/rucio/web/rest/flaskapi/v1/common.py

```diff
@@ -210,6 +210,15 @@ def parse_scope_name(scope_name: str, vo: Optional[str]) -> tuple[str, ...]:
     :raises ValueError: when scope_name could not be parsed.
     :returns: a (scope, name) tuple.
     """
+
+    encoded_slash = r"%2F"
+    if encoded_slash in scope_name:
+        if scope_name.count('/') != 1:
+            raise ValueError(f"Could not parse '{scope_name}' ({scope_name=}) with encoded '/' into scope and name.")
+        scope, name = scope_name.split('/', 1)
+        name = name.replace(encoded_slash, '/')
```

Comment:
It might be more appropriate to use `unquote_plus`, in order to do the complete inverse of the `quote_plus` operation done in the client:

https://docs.python.org/3/library/urllib.parse.html#urllib.parse.unquote_plus

Reply:
Updated.